app-admin/kubelet/files/kubelet

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Wrapper for launching kubelet via rkt-fly stage1
+
+set -e
+
+if [ -z "${KUBELET_VERSION}" ]; then
+    echo "ERROR: must set KUBELET_VERSION"
+    exit 1
+fi
+
+KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+
+mkdir --parents /etc/kubernetes
+mkdir --parents /var/lib/docker
+mkdir --parents /var/lib/kubelet
+mkdir --parents /run/kubelet
+
+exec /usr/bin/rkt run \
+  --volume etc-kubernetes,kind=host,source=/etc/kubernetes \
+  --volume etc-ssl-certs,kind=host,source=/usr/share/ca-certificates \
+  --volume var-lib-docker,kind=host,source=/var/lib/docker \
```

Comment:
Eventually, the kubelet will spawn rkt pods, and I think we something for rkt to work properly in rkt fly. Here are some thoughts.
We need `/etc/rkt` for various rkt and CNI configurations. We have possible conflicts with CNI if the kubelet and the host both execute rkt, so we need at least `/var/lib/cni` for the `host-local` allocator. 
Then there might be issues with GCing. If `/var/lib/rkt` will not be mounted the pods started by the kubelet are not going to be GCd by the host's GC timer. 
Note that it won't be mounted recursively and will not have overlayfs mounts etc. from the host's pods but that, but I'm not 100% sure about the consequences this will have if rkt GC would be called by the kubelet and only find the empty pod directories. @jonboulle @vcaputo maybe one of you can foresee this?
Where will the rkt API service be running? This is going to be tricky if the kubelet and the host both run pods and don't share resources examined by the API service. @yifan-gu